### ISSUE TITLE
Themes ghost updates

### ISSUE THREAD

**Issue by BlackSkorpio** [koken/issues#7]
In some cases Koken reports that there is themes in need of an update: even if those themes no longer is installed on the system.
![ghost-updates](https://cloud.githubusercontent.com/assets/11587201/12934466/110e168e-cf8f-11e5-9ac8-9ffa5b96f760.PNG)
![ghost](https://cloud.githubusercontent.com/assets/11587201/12934469/14856cb8-cf8f-11e5-8710-8a42c998593b.PNG)

**Comment by BigBobDalton** [comment]
yeah, I have the same problem with no clue as how to resolve.  I even manually deleted the theme from the update.php but that didn't help and I fear it is in the SQL database as updated until Koken writes complete
